pallets/asset-manager/src/lib.rs

```diff
@@ -166,6 +166,17 @@ pub mod pallet {
 		fn get_units_per_second(asset_type: T::ForeignAssetType) -> Option<u128> {
 			AssetTypeUnitsPerSecond::<T>::get(asset_type)
 		}
+		#[cfg(feature = "runtime-benchmarks")]
+		fn set_units_per_second(asset_type: T::ForeignAssetType, fee_per_second: u128) {
```

Comment:
Since this code is doing something very unusual, a quick comment / caution would be nice. The cfg flag is a nice hint already, though :)